rust/agama-cli/src/profile.rs

```diff
@@ -45,15 +56,55 @@ fn validate(path: String) -> anyhow::Result<()> {
 fn evaluate(path: String) -> anyhow::Result<()> {
     let evaluator = ProfileEvaluator {};
     evaluator
-        .evaluate(Path::new(&path))
+        .evaluate(Path::new(&path), stdout())
         .context("Could not evaluate the profile".to_string())?;
     Ok(())
 }
 
-pub fn run(subcommand: ProfileCommands) -> anyhow::Result<()> {
+async fn import(url: String, dir: Option<PathBuf>) -> anyhow::Result<()> {
```

Comment:
The other commands (`download` and `evaluate`) rely on a separate struct. If you do not want to build a struct for this, it is fine, but at least I would move the logic to the same separate module.

I would like to keep the functions here to handle the details that are specific to the commands (setting default values, etc.).